### ISSUE TITLE
Sudden blackness or ant other colors when using browsers

### ISSUE THREAD

**Issue by shamito** [elementary/switchboard-plug-display#263]
Not sure if you experienced this with the latest version of elementary OS 5.1.7 Hera, whenever watching youtube I am experiencing a sudden full screen cover of red and sometime using epiphany browser where scrolling sometimes a black screen appears. Not quite sure whats the problem was.

**Comment by tintou** [comment]
This looks like a driver issue, is it happening with any other operating system?

**Comment by shamito** [comment]
No, before installing elemOS, I am using win10 and never experience this.I just noticed that whenever a full screen cover happens it only covers the whole browser screen.
